code/datums/diseases/advance/symptoms/clockwork.dm

```diff
@@ -0,0 +1,436 @@
+					O.burn_reduction = max(4, O.burn_reduction)
+					O.brute_reduction = max(5, O.brute_reduction)
+				continue
+			switch(O.body_zone)
+				if(BODY_ZONE_HEAD)
+					var/obj/item/bodypart/head/robot/clockwork/B = new()
+					if(robustbits)
+						B.brute_reduction = 5
+						B.burn_reduction = 4
+					B.replace_limb(H, TRUE)
+					H.visible_message("<span_class='userdanger'>Your head feels numb, and cold.</span>")
+					qdel(O)
+					return TRUE
+				if(BODY_ZONE_CHEST)
+					var/obj/item/bodypart/chest/robot/clockwork/B = new()
+					if(robustbits)
+						B.brute_reduction = 5
+						B.burn_reduction = 4
+					B.replace_limb(H, TRUE)
+					H.visible_message("<span_class='userdanger'>Your [O] feels numb, and cold.</span>")
+					qdel(O)
+					return TRUE
+				if(BODY_ZONE_L_ARM)
+					var/obj/item/bodypart/arm/left/robot/clockwork/B = new()
+					if(robustbits)
+						B.brute_reduction = 5
+						B.burn_reduction = 4
+					B.replace_limb(H, TRUE)
+					H.visible_message("<span_class='userdanger'>Your [O] feels numb, and cold.</span>")
+					qdel(O)
+					return TRUE
+				if(BODY_ZONE_R_ARM)
+					var/obj/item/bodypart/arm/right/robot/clockwork/B = new()
+					if(robustbits)
+						B.brute_reduction = 5
+						B.burn_reduction = 4
+					B.replace_limb(H, TRUE)
+					H.visible_message("<span_class='userdanger'>Your [O] feels numb, and cold.</span>")
+					qdel(O)
+					return TRUE
+				if(BODY_ZONE_L_LEG)
+					var/obj/item/bodypart/leg/left/robot/clockwork/B = new()
+					if(robustbits)
+						B.brute_reduction = 5
+						B.burn_reduction = 4
+					B.replace_limb(H, TRUE)
+					H.visible_message("<span_class='userdanger'>Your [O] feels numb, and cold.</span>")
+					qdel(O)
+					return TRUE
+				if(BODY_ZONE_R_LEG)
+					var/obj/item/bodypart/leg/right/robot/clockwork/B = new()
+					if(robustbits)
+						B.brute_reduction = 5
+						B.burn_reduction = 4
+					B.replace_limb(H, TRUE)
+					H.visible_message("<span_class='userdanger'>Your [O] feels numb, and cold.</span>")
+					qdel(O)
+					return TRUE
+	return FALSE
+
+/datum/symptom/robotic_adaptation/End(datum/disease/advance/A)
+	if(!..())
+		return
+	var/mob/living/carbon/human/H = A.affected_mob
+	//REMOVE_TRAIT(H, TRAIT_NANITECOMPATIBLE, DISEASE_TRAIT)
+	if(A.stage >= 5 && (replaceorgans || replacebody)) //sorry. no disease quartets allowed
+		to_chat(H, "<span class='userdanger'>You feel lighter and springier as your innards lose their clockwork facade.</span>")
+		H.dna.species.regenerate_organs(H, replace_current = TRUE)
+		for(var/obj/item/bodypart/O in H.bodyparts)
+			if(!IS_ORGANIC_LIMB(O))
+				O.burn_reduction = initial(O.burn_reduction)
+				O.brute_reduction = initial(O.brute_reduction)
+
+/datum/symptom/robotic_adaptation/OnRemove(datum/disease/advance/A)
+	A.infectable_biotypes -= MOB_ROBOTIC
+
+//below this point lies all clockwork bits that make this symptom tick. no pun intended.
+/obj/item/organ/internal/ears/robot/clockwork
```

Comment:
these should be seperated out into their respective modular folders for consistancy

Reply:
i don't know what you mean by modular folder.

Reply:
we have a folder called monkestation, move it to the same file name in their, if it doesn't exist just create the file that way